storm-core/test/clj/backtype/storm/config_test.clj

```diff
@@ -168,3 +168,24 @@
                       (catch Exception e e)))))
         (is (thrown-cause? java.lang.IllegalArgumentException
           (.validateField validator "test" 42)))))))
+
+(deftest test-absolute-storm-local-dir
+  (let [storm-home-key "storm.home"
+        old-storm-home (System/getProperty storm-home-key)
+        conf-relative {STORM-LOCAL-DIR "storm-local"}
+        conf-absolute {STORM-LOCAL-DIR
+                       (if on-windows?
+                         "C:\\storm-local"
+                         "/var/storm-local")}]
+    (try
+      (System/setProperty storm-home-key (if on-windows? "C:\\storm-home" "/usr/local/storm-home"))
```

Comment:
@Renkai 
Unfortunately I found this PR affects other tests randomly - supervisor tests are failing. 
Please see https://travis-ci.org/apache/storm/jobs/79231189.
Could we just use current storm.home?